test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestSettingsContext.cs

```diff
@@ -137,5 +138,52 @@ public static void RemoveSetting(XDocument doc, string key)
                 item.Remove();
             }
         }
+
+        // Add NetStandard.Library and NetCorePlatforms to the feed and save the file.
+        public void AddNetStandardFeeds()
+        {
+            var reposRoot = GetRepositoryRootDirectory();
+            var netStandardLibraryPackageFeed = GetNetStandardLibraryFeed(reposRoot);
+            var netCorePlatformsPackageFeed = GetNetCorePlatformFeed(reposRoot);
+
+            Assert.True(Directory.Exists(netStandardLibraryPackageFeed));
+            Assert.True(Directory.Exists(netCorePlatformsPackageFeed));
+
+            var section = GetOrAddSection(XML, "packageSources");
+            AddEntry(section, "NetStandardLibrary", netStandardLibraryPackageFeed);
+            AddEntry(section, "NetCorePlatforms", netCorePlatformsPackageFeed);
+
+            Save();
+        }
+
+        private static DirectoryInfo GetRepositoryRootDirectory()
+        {
+            DirectoryInfo currentDir = new DirectoryInfo(Directory.GetCurrentDirectory());
+            ;            while (currentDir != null)
+            {
+                if (File.Exists(Path.Combine(currentDir.FullName, "NuGet.sln")))
+                {
+                    // We have found the repo root.
+                    break;
```

Comment:
I think it would be better to return here, when found, and throw a directory not found exception at the end of the method, rather than returning null, when not found.

Reply:
Thanks! Changed.